packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -161,7 +166,7 @@ contract DeputyPauseModule is ISemver, EIP712 {
     /// @param _signature ECDSA signature.
     function pause(bytes32 _nonce, bytes memory _signature) external {
         // Verify the signature.
-        bytes32 digest = _hashTypedDataV4(keccak256(abi.encode(PAUSE_MESSAGE_TYPEHASH, PauseMessage(_nonce))));
+        bytes32 digest = _hashTypedDataV4(keccak256(abi.encode(PAUSE_MESSAGE_TYPEHASH, _nonce)));
```

Comment:
I don't know enough to have an opinion, but why is this better?

Reply:
Basically the original approach was not spec compliant with EIP-712. It happened to work for this case but you aren't really supposed to do it this way.